python_modules/dagster/dagster/cli/run.py

```diff
@@ -11,9 +12,17 @@ def create_run_cli_group():
 
 
 @click.command(name="list", help="List the runs in this dagster installation.")
-def run_list_command():
+@click.option(
+    "--limit", help="Only list a specified number of instances", is_flag=True, type=int,
+)
+def run_list_command(limit):
     with DagsterInstance.get() as instance:
-        for run in instance.get_runs():
+        if limit:
+            runs = islice(instance.get_runs(), limit)
+        else:
+            runs = instance.get_runs()
```

Comment:
The `get_runs` method actually takes a `limit` argument, which includes the limit in the DB query itself for getting runs. We should probably use that instead of doing the splice over the entire set of runs.